drivers/spi/spi_nrfx_spim.c

```diff
@@ -90,6 +91,9 @@ static inline void finalize_spi_transaction(const struct device *dev, bool deact
 	if (NRF_SPIM_IS_320MHZ_SPIM(reg) && !(dev_data->ctx.config->operation & SPI_HOLD_ON_CS)) {
 		nrfy_spim_disable(reg);
 	}
+	if (IS_ENABLED(CONFIG_PM_DEVICE_RUNTIME)) {
```

Comment:
This is unneeded. `pm_device_runtime_put()` can be called without enabled `PM_DEVICE_RUNTIME`. Same goes for `pm_device_runtime_get()`.

Reply:
removed redundant conditionals